### DISCUSSION TITLE
How to add a custom domain name?

### DISCUSSION THREAD

**Discussion by thomasofdenmark** [Azure/azure-dev#2748] [Q&A]
I have used the `todo-nodejs-mongo-aks` template to scaffold my app. How do I set up a custom domain name in the bicep files?

**Comment by savannahostrowski** [comment]
I think this is an area which would be great for us to expand our docs on (cc: @gkulin).

@wbreza any guidance you'd provide here? I have experience with custom domains and ACA but not AKS.
